packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3217,14 +3217,14 @@ export class ContainerRuntime
 			if (runGC) {
 				await this.collectGarbage(
 					{ logger: summaryLogger, runSweep, fullGC },
-					telemetryContext,
+					telemetryContext as ITelemetryContext,
 				);
 			}
 
 			const { stats, summary } = await this.summarizerNode.summarize(
 				fullTree,
 				trackState,
-				telemetryContext,
+				telemetryContext as ITelemetryContext,
```

Comment:
Why is this (and the other) cast necessary?

Reply:
No longer needed after remove get()